examples/pages/empty.tsx

```diff
@@ -15,6 +23,54 @@ const LANGUAGES: Options['languages'] = [
   },
 ];
 
+function Toolbar() {
+  const insert = useInsertNew();
+  const setIsPreviewMode = useSetPreviewMode();
+  const setEditMode = useSetEditMode();
+
+  function addSlate() {
+    insert({ plugin: 'ory/editor/core/content/slate' });
+  }
+
+  function previewMode() {
+    setIsPreviewMode();
+  }
+
+  function editMode() {
+    setEditMode();
+  }
+
+  return (
+    <div style={{ padding: '20px' }}>
+      <Button
+        onClick={addSlate}
+        startIcon={<Add />}
+        color={'primary'}
+        variant={'contained'}
+        style={{ marginRight: '10px' }}
+      >
+        Add slate
+      </Button>
+      <Button
+        onClick={previewMode}
+        color={'primary'}
+        variant={'contained'}
+        style={{ marginRight: '10px' }}
+      >
+        Preview Mode
+      </Button>
+      <Button
+        onClick={editMode}
+        color={'primary'}
+        variant={'contained'}
+        style={{ marginRight: '10px' }}
+      >
+        Edit Mode
+      </Button>
+    </div>
+  );
+}
+
```

Comment:
please remove this from the "empty" example

Reply:
Will do.